kdl_parser/package.xml

```diff
@@ -27,11 +27,12 @@
   <build_depend>roscpp</build_depend>
   <build_depend>urdf</build_depend>
   <build_depend>cmake_modules</build_depend>
-  <build_depend>rostest</build_depend>
 
-  <run_depend version_gte="1.3.0">orocos_kdl</run_depend>
-  <run_depend>rosconsole</run_depend>
-  <run_depend>roscpp</run_depend>
-  <run_depend>urdf</run_depend>
+  <exec_depend version_gte="1.3.0">orocos_kdl</exec_depend>
+  <exec_depend>rosconsole</exec_depend>
+  <exec_depend>roscpp</exec_depend>
+  <exec_depend>urdf</exec_depend>
```

Comment:
urdf being part of the public api, should it be also an exported build dependency ? (same applies to orocos_kdl)
Looks like the dependency on tinyxml is missing as well